src/scss/2021/components/_bar-chart.scss

```diff
@@ -46,13 +46,13 @@
     color: var(--color);
     display: flex;
   }
-  
+
   &__value {
     display: inline;
     @include type-mono-micro();
-    line-height: 1;
-    padding: 0.125rem;
-    margin-bottom: 0.5rem;
+    line-height: 1rem;
```

Comment:
`line-height` ideally should not have units, it’s based on a relative ratio to the font-size.